.env

```diff
@@ -0,0 +1 @@
+VITE_WEB_TITLE = WEB TITLE
```

Comment:
Never commit a env file up to the repo. 

Env files can contain sensitive such as passwords and keys, so this file actually needs to be included in `.gitignore`.

That being said, it's important to include an example of what this file should look like. 

Create a new file, with placeholder values, called `.env.example`. 

This file can be safely committed to the repo, since it does not contain actual values.

This way, anyone who clones this repo can have their own `.env` file, which is automatically ignored by default. We are helping protect any future user.